galsim/chromatic.py

```diff
@@ -2740,8 +2844,10 @@ def __init__(self, obj, gsparams=None, propagate_gsparams=True):
                                                          self._propagate_gsparams)
         else:
             self.deinterpolated = self
-        self.SED = SED(lambda w: 1./obj.SED(w), 'nm', '1')
-        self.wave_list = obj.wave_list
+
+    @property
+    def sed(self):
```

Comment:
Shouldn't deconvolved_sed = 1/original_sed like the original?  Seems like there may be a missing round-trip sort of unit test here.